src/server/lib/recaptcha.ts

```diff
@@ -1,74 +1,94 @@
 import { NextFunction, Request, Response } from 'express';
-import { RecaptchaV2 } from 'express-recaptcha';
 import { CaptchaErrors } from '@/shared/model/Errors';
 import { getConfiguration } from '@/server/lib/getConfiguration';
 import { logger } from '@/server/lib/serverSideLogger';
 import { trackMetric } from '@/server/lib/trackMetric';
 import { HttpError } from '@/server/models/Error';
-import { featureSwitches } from '@/shared/lib/featureSwitches';
```

Comment:
It's also worth removing the `recaptchaEnabledDev` key from the features switches object.

Reply:
https://github.com/guardian/gateway/blob/main/src/shared/lib/featureSwitches.ts